vulture_os/toolkit/api_parser/cortex_xdr/cortex_xdr.py

```diff
@@ -29,21 +29,16 @@
 
 from datetime import datetime
 from django.conf import settings
-from django.utils import timezone
 from django.utils.translation import gettext_lazy as _
+from pytz import utc
+
```

Comment:
Why do we need a change there?
Does the django.utils' timezone pose any problem?